### DISCUSSION TITLE
show tg-usernames on discord in bold and ...

### DISCUSSION THREAD

**Discussion by renewid** [42wim/matterbridge#1651] [Ideas]
... discord-usernames on tg bold by default, too. 1st is already done, but on tg its only plain text.
or do sth similar to make messages on tg more readable.

**Comment by tahmeaollo** [comment]
Indeed at the moment discord discussions on telegram side are totally unreadable...

I would also suggest that there would be a line break after bold usernames. Also I don't think that there is a need for the "(discord)" in before usernames as the bot name already tells that it's from discord.
